lib/template.html

```diff
@@ -108,6 +108,10 @@
   <meta property="og:title" content="{{ seo_page_title }}" />
 {% endif %}
 
+{% if seo_author %}
+  <meta name="author" content="{{ seo_author }}" />
+{% endif %}
```

Comment:
In the case `seo_author` is a key which maps to an item in `site.data.authors.yml` we should probably make the author name output using `site.data.authors[seo_author].name`.